config/sysroot.gni

```diff
@@ -54,11 +54,10 @@ if (current_os == target_os && current_cpu == target_cpu &&
     } else if (_script_arch == "x64") {
       _script_arch = "amd64"
     }
-    assert(
-        exec_script("//build/dir_exists.py",
-                    [ rebase_path(sysroot) ],
-                    "string") == "True",
-        "Missing sysroot ($sysroot). To fix, run: build/linux/sysroot_scripts/install-sysroot.py --arch=$_script_arch")
+    if (exec_script("//build/dir_exists.py", [ rebase_path(sysroot) ], "string") != "True") {
```

Comment:
Is this something that we'd be able to push back to google? If not, we will need to change this...

Reply:
That is added because of broken preparation for Linux x86. Not sure if this fix is still required.

Reply:
Can you see if "official" has resolved this issue?